### DISCUSSION TITLE
v3.0.5-beta

### DISCUSSION THREAD

**Discussion by mrghozzi** [mrghozzi/myads#37] [Announcements]
Upcoming updates will be dedicated to a set of fixes (please report any bug in the script).
    After making sure that the errors are fixed, we will start working on the stable version v3.1.0 with some additions

**Fix**: Message template and add `reloadChat()` to even see incoming messages without resorting to reloading the page.
**Fix**: Wrongly translated dates.
**Improvement**: List of ad banners.
**Fix**: Error with writing editor in store knowledge base.
**Fix**: Error when a white page appears when entering a non-existent member page.
**Fix**: The problem of entering a member profile whose name consists of numbers only.
**Add**: Earn points by posting in the community and participating in posts and comments.
**Add**: Stats Box widgets.
**Fix**: Problem showing posts.

**Comment by bladhac** [comment]
In the next version, I recommend RTL support

**Reply by mrghozzi** [reply]
We will try to work on it in the next versions

**Comment by alaa-eg** [comment]
**- تعديل ملف اللغة العربية في التحديث القادم**

```
$lang['no_com']      = "لا يوجد تعليقات";
$lang['no_post']     = "لا يوجد مقالات جديدة";

$lang['cat']         = "التصنيف :";

$lang['t_cat']       = "جميع التصنيفات";
$lang['e_cat']       = "تعديل التصنيف";

$lang['no_user']     = "لا يوجد أعضاء";

$lang['t_comm']      = "مجموع التعليقات";

$lang['n_viset']     = "عدد الزائرين الآن";

$lang['no_page']     = "لا يوجد صفحات";

$lang['towthbaner']  = "لمشاهدة البنرات الإعلانية";

$lang['sieanpr']     = "المتجر فارغ ، أضف منتج جديد";

$lang['create_a']    = "إنشاء";

$lang['lastps']      = "آخر نشر";

$lang['second']      = "ثانية";

$lang['lastcontact'] = "آخر إتصال منذ";
```

**Reply by alaa-eg** [reply]
**- تعديل ملف اللغة الإنجليزية في التحديث القادم**

```
$lang['e_mail']      = "E-mail address :";
$lang['email']       = "E-mail address";

$lang['donthaacc']   = "Don't have an account?";
```

**Reply by mrghozzi** [reply]
**شكرا على الملاحظات.**
تم إجراء الإصلاحات القترحة, ستكون متاحة في التحديث القادم.
